### PR TITLE
use path module instead of regex and string concat

### PR DESCRIPTION
**Pull request by ixtli**

### CODE REVIEW

File: es6-website/Brocfile.js [hunk 1]
```diff
@@ -11,11 +14,12 @@ var mergeTrees = require('broccoli-merge-trees');
 var appJs = babel('src');
 
 // Grab the polyfill file provided by the Babel library
+// Grab the polyfill file provided by the Babel library
 var babelPath = require.resolve('broccoli-babel-transpiler');
-babelPath = babelPath.replace(/\/index.js$/, '');
-babelPath += '/node_modules/babel-core';
+babelPath = path.dirname(babelPath);
+babelPath = path.join(babelPath, '/node_modules/babel-core');
```
Comment (stefanpenner) [inline es6-website/Brocfile.js]: leave off the '/' instead use join for each step

File: es6-website/Brocfile.js [hunk 2]
```diff
@@ -11,11 +14,12 @@ var mergeTrees = require('broccoli-merge-trees');
...
 var browserPolyfill = funnel(babelPath, {
-  files: ['browser-polyfill.js']
+	files: ['browser-polyfill.js']
```
Comment (givanse) [inline es6-website/Brocfile.js]: indent too big

File: es6-website/Brocfile.js [hunk 3]
```diff
@@ -11,11 +14,12 @@ var mergeTrees = require('broccoli-merge-trees');
 var appJs = babel('src');
 
 // Grab the polyfill file provided by the Babel library
+// Grab the polyfill file provided by the Babel library
```
Comment (givanse) [inline es6-website/Brocfile.js]: duplicated comment

**Comment by ixtli** [comment]
Fixed indent, deleted c/p redundancy, and passed multiple arguments to join. Thanks for the feedback :)

**Pull request merged**